src/components/pages/GroomerDashboard/groomer-tabs.js

```diff
@@ -0,0 +1,77 @@
+import { Tabs, Row } from 'antd';
```

Comment:
It looks like 'Row' may or maynot be used later when we add content to the tabs, so for now just go ahead and remove 'Row' from the import so that the warning will be removed.